nextflow.config

```diff
@@ -17,10 +17,68 @@ params {
     file_prefix_in = ''
     file_prefix_out = ''
     version = '4.2.1'
+    // Docker containers reused across processes
+    container_cellprofiler = "cellprofiler/cellprofiler:${params.version}"
+    container_pandas = "quay.io/fhcrc-microbiome/python-pandas:v1.2.1_latest"
 }
 
+
+profiles {
+    standard {
+        process {
+            withLabel: io_limited {
+                cpus = 1
+                memory = 4.GB
+            }
+            withLabel: mem_medium {
+                cpus = 4
+                memory = 30.GB
+            }
+            withLabel: cpu_medium {
+                cpus = 8
+                memory = 15.GB
+            }
+            withLabel: cpu_verylarge {
+                cpus = 36
+                memory = 58.GB
+            }
+            errorStrategy = 'retry'
+            maxRetries = 5
+        }
+    }
+    docker {
+        docker {
+            enabled = true
+            temp = 'auto'
+        }
+        process {
+            withLabel: io_limited {
+                cpus = 1
+                memory = 1.GB
+            }
+            withLabel: mem_medium {
+                cpus = 1
+                memory = 1.GB
+            }
+            withLabel: cpu_medium {
+                cpus = 1
+                memory = 1.GB
+            }
+            withLabel: cpu_verylarge {
+                cpus = 1
+                memory = 1.GB
+            }
+            errorStrategy = 'retry'
+            maxRetries = 5
+            maxForks = 10
```

Comment:
I don't think we should limit the rate of execution to 10 concurrent processes by default

Reply:
Ah, sorry. This was a remnant from some testing; I'll fix it.